modules/datahandling/filerequests_test.go

```diff
@@ -258,8 +251,9 @@ func TestFileChangeRequest_Process(t *testing.T) {
 	req.Resource = "File"
 	req.Method = "Change"
 	req.FileID = fileid
-	req.Changes = []string{"change 1"}
-	req.BaseFileVersion = newFileVersion
+	req.Changes = []string{"v0:\n0:+1:a"}
+
+	baseFileVersion := int64(1)
 
```

Comment:
I think that was meant to be on a separate line; it's initializing different parts of the environment. I'm not sure though; @jshap70 might know more about this one